servers/quarkus-server/src/main/java/com/dremio/nessie/server/config/ApplicationConfig.java

```diff
@@ -126,4 +133,14 @@ public VersionStoreDynamoConfig getVersionStoreDynamoConfig() {
     @ConfigProperty(name = "valueTableName", defaultValue = "nessie_objects")
     String getValueTableName();
   }
+
+  @ConfigProperties(prefix = "nessie.version.store.mongodb")
+  public interface VersionStoreMongoDbConfig {
```

Comment:
Please add the table names as options here.

Reply:
The wiring into nessie has been removed as previously discussed.